activejob/lib/active_job/queue_priority.rb

```diff
@@ -18,7 +18,25 @@ module ClassMethods
       #     end
       #   end
       #
-      # Specify either an argument or a block.
+      # Can be given a block that will evaluate in the context of the job
+      # allowing +self.arguments+ to be accessed so that a dynamic priority
+      # can be applied:
```

Comment:
This isn't only about arguments, you can use the block to check anything about the job you want. So the following would be more correct.

```suggestion
      # Can be given a block that will evaluate in the context of the job
      # so that a dynamic priority can be applied:
```

Reply:
Thanks for the review. I've made the suggested change.

Should we also make the same change for [queue_as](https://api.rubyonrails.org/classes/ActiveJob/QueueName/ClassMethods.html#method-i-queue_as) which uses the same wording as above? If yes, I can raise another PR for that.

Reply:
Sounds good!